### PR TITLE
feat(off-chain): Optimize the multi-signature, making it possible to upgrade seamlessly

### PR DESCRIPTION
**Pull request by LeonLi000**

### CODE REVIEW

File: ckb-contracts/types/schemas/eth_bridge_lock_cell.mol [hunk 1]
```diff
@@ -3,4 +3,5 @@ import basic;
 struct ETHBridgeLockArgs {
     eth_contract_address: ETHAddress,
     eth_token_address: ETHAddress,
+    typescript_hash: Byte32,
```
Comment (huwenchao) [inline ckb-contracts/types/schemas/eth_bridge_lock_cell.mol:6]: use `light_client_typescript_hash` to make the name more clear.

File: eth-contracts/contracts/TokenLocker.sol [hunk 2]
```diff
@@ -96,7 +100,8 @@ contract TokenLocker {
             bridgeFee,
             recipientLockscript,
             replayResistOutpoint,
-            sudtExtraData
+            sudtExtraData,
+            bridgeLockscript
```
Comment (huwenchao) [inline eth-contracts/contracts/TokenLocker.sol:104]: Why is this needed?

**Review by Yedfree — APPROVED** [review]
